includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -186,7 +186,7 @@ public function pull( $items ) {
 
 			if ( ! empty( $post_props['meta']['dt_connection_map'] ) ) {
 				foreach ( $post_props['meta']['dt_connection_map'] as $distributed ) {
-					$distributed = unserialize( $distributed );
+					$distributed = unserialize( $distributed ); // @codingStandardsIgnoreLine valid use of unserialize
```

Comment:
Would be good to have an explanation about why this can't be `maybe_unserialize()`, which is supposed to be safer.